testdata/nested-structs.go

```diff
@@ -0,0 +1,13 @@
+package fixtures // MATCH /no nested structs are allowed, got 1/
+
+type Foo struct {
+	Bar struct {
```

Comment:
As user I expect a failure referring to this line thus the test should read as:

```go
type Foo struct {
  Bar struct { // MATCH /no nested structs are allowed/
  }
}
```

Reply:
Oh I didn't know you could match on individual lines. Nice, I'll fix that.